### PR TITLE
feat: add chat theme package

### PR DESCRIPTION
**Pull request by rileymiller**

### CODE REVIEW

File: stream-projects/chat-theme/shared/extension/signals.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {
+  getConnectionSourceType as getChannelSourceType,
+  useComputed,
+  useExtensionInfo$,
+} from "../../deps.ts";
+
+export function useSourcetype$() {
```
Comment (austinhallock) [inline stream-projects/chat-theme/shared/extension/signals.ts:7]: useSourceType

File: stream-projects/chat-theme/shared/alerts/signals.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+`;
+
+export function useAlertSubscription$(
+  { status, types, limit }: {
+    status?: string;
+    types?: string[];
+    limit?: number;
+  },
+) {
+  const { signal$: alertConnection$ } = useSubscriptionSignal(
```
Comment (austinhallock) [inline stream-projects/chat-theme/shared/alerts/signals.ts:33]: we should standardize between ___Signal() vs ____$()

File: stream-projects/chat-theme/public/global.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+html, body, #ultra, main {
```
Comment (austinhallock) [inline stream-projects/chat-theme/public/global.css:1]: can rm public folder (doesn't get used)

File: stream-projects/chat-theme/deps.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  useSignal,
+  useSubscriptionSignal,
+} from "https://tfl.dev/@truffle/state@~0.0.8/mod.ts";
+export { gql } from "https://tfl.dev/@truffle/api@~0.1.19/mod.ts";
+export {
+  Memo,
+  useComputed,
+  useSelector,
+} from "https://npm.tfl.dev/@legendapp/state@~0.19.0/react";
+export type { TruffleGQlConnection } from "https://tfl.dev/@truffle/api@^0.1.0/types/mod.ts";
```
Comment (austinhallock) [inline stream-projects/chat-theme/deps.ts:22]: ~0.2.0

File: stream-projects/chat-theme/deps.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  useState,
+} from "https://npm.tfl.dev/react";
+export { default as jumper } from "https://tfl.dev/@truffle/utils@~0.0.3/jumper/jumper.ts";
+export { useStyleSheet } from "https://tfl.dev/@truffle/distribute@^2.0.0/format/wc/react/index.ts"; // DO NOT BUMP;
+export { default as scss } from "https://tfl.dev/@truffle/utils@~0.0.3/css/css.ts";
+export {
+  useSignal,
+  useSubscriptionSignal,
+} from "https://tfl.dev/@truffle/state@~0.0.8/mod.ts";
+export { gql } from "https://tfl.dev/@truffle/api@~0.1.19/mod.ts";
```
Comment (austinhallock) [inline stream-projects/chat-theme/deps.ts:16]: ~0.2.0

File: stream-projects/chat-theme/shared/alerts/signals.ts (hunk 2 above)
Comment (rileymiller) [reply stream-projects/chat-theme/shared/alerts/signals.ts:33]: yeah that's a good point, is seems like we usually use ` ___Signal()` for shared utils/hooks but `___$()` for instances but it gets a little grey for custom hooks.

**Pull request merged**